### PR TITLE
Added overview report.

### PR DESCRIPTION
**Pull request by Kamulau**

### CODE REVIEW

**Comment by Kamulau** [comment]
Added AL2 headers to close #4

File: src/main/java/org/ltr4l/Main.java [hunk 1]
```diff
@@ -1,3 +1,19 @@
+/**
```
Comment (kojisekig) [inline src/main/java/org/ltr4l/Main.java]: Kamu,

Don't include the different issue for a PR of "Add Overview report." Make this in the separate PR.

File: src/main/java/org/ltr4l/Main.java [hunk 2]
```diff
@@ -1,3 +1,19 @@
...
+ * Copyright [yyyy] [name of copyright owner]
```
Comment (kojisekig) [inline src/main/java/org/ltr4l/Main.java]: And this looks strange. The copyright line should look like:

> Copyright 2018 org.LTR4L
